### PR TITLE
0.12.2

### PR DESCRIPTION
**Pull request by RemagOfficial**
updated forge
fixed pack version
updated a whole bunch of mods (client/server)
added ReAuth (client)

### CODE REVIEW

File: overrides/config/forge.cfg [hunk 1]
```diff
@@ -13,6 +13,9 @@ client {
     # Enable forge to queue all chunk updates to the Chunk Update thread. May increase FPS significantly, but may also cause weird rendering lag. Not recommended for computers without a significant number of cores available.
     B:alwaysSetupTerrainOffThread=false
 
+    # Allow item rendering to detect emissive quads and draw them properly. This allows glowing blocks to look the same in item form, but incurs a very slight performance hit.
+    B:allowEmissiveItems=true
```
Comment (Merlingilb) [inline overrides/config/forge.cfg:17]: @RemagOfficial Isn't this a permormance problem?

File: overrides/config/jei/worldSettings.cfg [hunk 2]
```diff
@@ -2,4 +2,53 @@
 
 ~CONFIG_VERSION: 0.1.0
 
+world-320001708 {
```
Comment (Merlingilb) [inline overrides/config/jei/worldSettings.cfg]: @RemagOfficial these lines are settings for specific worlds and as such not needed in the pack

File: overrides/config/journeymap_ModInfo.cfg [hunk 3]
```diff
@@ -3,8 +3,8 @@
 // For more information: https://github.com/MCModInfo/modinfo/blob/master/README.md
 {
   "modId": "journeymap",
-  "enable": true,
```
Comment (Merlingilb) [inline overrides/config/journeymap_ModInfo.cfg:6]: @RemagOfficial why would you disable ModInfo?

File: overrides/config/journeymap_server.cfg [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Configuration file
```
Comment (Merlingilb) [inline overrides/config/journeymap_server.cfg:1]: @RemagOfficial This file won't be needed for a client pack

File: overrides/config/fancymenu/customization/GuiWorldSelection.txt [hunk 5]
```diff
@@ -0,0 +1,144 @@
+type = menu
```
Comment (Merlingilb) [inline overrides/config/fancymenu/customization/GuiWorldSelection.txt:1]: waiting for new files

File: overrides/config/fancymenu/customization/GuiMainMenu.txt [hunk 6]
```diff
@@ -0,0 +1,245 @@
+type = menu
```
Comment (Merlingilb) [inline overrides/config/fancymenu/customization/GuiMainMenu.txt:1]: waiting for new files

File: overrides/config/fancymenu/customizablemenus.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+type = customizablemenus
```
Comment (Merlingilb) [inline overrides/config/fancymenu/customizablemenus.txt:1]: needs update

**Review by Merlingilb — CHANGES_REQUESTED** [review]

File: overrides/config/forge.cfg (hunk 1 above)
Comment (RemagOfficial) [reply overrides/config/forge.cfg:17]: "May increase FPS significantly"

**Pull request closed without merging**